test/e2e/keycloaks_test.go

```diff
@@ -311,7 +311,7 @@ func prepareKeycloaksCR(t *testing.T, f *framework.Framework, ctx *framework.Con
 
 func prepareKeycloaksCRWithExtension(t *testing.T, f *framework.Framework, ctx *framework.Context, namespace string) error {
 	keycloakCR := getKeycloakCR(namespace)
-	keycloakCR.Spec.Extensions = []string{"https://github.com/aerogear/keycloak-metrics-spi/releases/download/1.0.4/keycloak-metrics-spi-1.0.4.jar"}
+	keycloakCR.Spec.Extensions = []string{"https://github.com/aerogear/keycloak-metrics-spi/releases/download/2.5.3/keycloak-metrics-spi-2.5.3.jar"}
```

Comment:
Could you please remove this change? It is already addressed by #432.

Reply:
:+1: